### DISCUSSION TITLE
Proposal: A001-ecosystem-tools-integration

### DISCUSSION THREAD

**Discussion by yufeiminds** [GuanceCloud/datakit#16] [Ideas]
A001: Ecosystem Tools Integration
----
* Author(s): 
* Approver: 
* Status: Draft
* Last updated: 2023-04-25
* Discussion at:

## Abstract

[A short summary of the proposal.]

## Background

[An introduction of the necessary background and the problem being solved by the proposed change.]

### Related Proposals: 
* A list of proposals this proposal builds on or supersedes.

## Proposal

[A precise statement of the proposed change.]

### Temporary environment variable protection

[Name the environment variable(s) used to enable/disable the feature(s) this proposal introduces and their default(s).  Generally, features that are enabled by I/O should include this type of control until they have passed some testing criteria, which should also be detailed here.  This section may be omitted if there are none.]

## Rationale

[A discussion of alternate approaches and the trade offs, advantages, and disadvantages of the specified approach.]

## Implementation

[A description of the steps in the implementation, who will do them, and when. If a particular language is going to get the implementation first, this section should list the proposed order.]

## Open issues (if applicable)

[A discussion of issues relating to this proposal for which the author does not know the solution. This section may be omitted if there are none.]

**Comment by qingoba** [comment]
Hello, will this proposal be updated soon? I am very interested in Go language and WasmEdge and am currently working on a WasmEdge-related project. I hope to join this project. Can I get in touch with you for further discussion？

**Reply by yufeiminds** [reply]
@qingoba This is only a template for drafting a proposal. You can contact mentor @coanor to apply, and we will help you to draft the application.
